### PR TITLE
created Sponsors image section

### PR DESCRIPTION
**Pull request by zidano88**
Created the new branch and the required sections

### CODE REVIEW

File: src/components/EvComponents/SponsorImage.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+const SponsorImage = (props) => {
+    return (
+        <Box sx={{
+            textAlign: "center",
+            display: "flex",
+            justifyContent: "center",
+            alignItems: "center",
+            margin: "10px"
+        }}>
+            <img src={props.imageSrc}
```
Comment (7ganam) [inline src/components/EvComponents/SponsorImage.jsx]: don't use vanilla img .. use the image compoennt from 
import Image from "components/BazarImage";

File: src/components/EvComponents/SponsorImage.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import { Box } from "@mui/system";
+import React from "react";
+
+const SponsorImage = (props) => {
+    return (
+        <Box sx={{
+            textAlign: "center",
+            display: "flex",
+            justifyContent: "center",
+            alignItems: "center",
```
Comment (7ganam) [inline src/components/EvComponents/SponsorImage.jsx]: give the box width and height of 150 px and make the image inside it have width 100% and height auto
This is to make sure each logo takes the same space
make sure it stays centered on small screens

File: src/components/EvSections/SponsorsImageGridSection.jsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    {
+        source: "/assets/images/brands/amazon.png",
+    },
+]
+
+const SponsorsImageGrid = () => {
+    return (
+        <Box sx={{ textAlign: "center", }}>
+            <SectionTitle>Event Sponsors</SectionTitle>
+            <Grid container columns={{ xs: 1, sm: 3, md: 6 }} sx={{ padding: "35px", textAlign: "center", placeItems: "center", }}>
```
Comment (7ganam) [inline src/components/EvSections/SponsorsImageGridSection.jsx]: this is not the right way to use the grid.... leave the container grid with 12 columns all the time
and set the item grids to {xs:12 , sm:4 , md:2}
contact me if you don't understand this point

**Review by 7ganam — CHANGES_REQUESTED** [review]

File: src/components/EvComponents/SponsorImage.jsx (hunk 1 above)
Comment (zidano88) [reply src/components/EvComponents/SponsorImage.jsx]: solved

File: src/components/EvComponents/SponsorImage.jsx (hunk 2 above)
Comment (zidano88) [reply src/components/EvComponents/SponsorImage.jsx]: solved

File: src/components/EvSections/SponsorsImageGridSection.jsx (hunk 3 above)
Comment (zidano88) [reply src/components/EvSections/SponsorsImageGridSection.jsx:56]: image

Comment (zidano88) [reply src/components/EvSections/SponsorsImageGridSection.jsx:56]: how can I solve this overlap?

Comment (zidano88) [reply src/components/EvSections/SponsorsImageGridSection.jsx]: solved

**Pull request merged**